packages/@okta/vuepress-site/docs/concepts/third-party-cookies/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Cross-Site Cookie Blocking
+---
+
+# Cross-Site Cookie Blocking
+
+Cross-site cookie blocking is a privacy-protection feature that is being added to web browsers like Safari and Firefox. It's likely to become common across most browsers in the future. It can disrupt certain customer-hosted flows in Okta.
+
+## Description of the problem
+
+Beginning from version 13.1, Safari [blocks third-party cookies by default](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more/), disrupting Okta functionality in certain flows. Firefox and other browsers are in the process of rolling out a similar change.
+
+This issue will not affect your organization if you use the Okta-hosted sign-in page. Only customers who host their own sign-in functionality can be affected.
```

Comment:
It seems like this paragraph contradicts information in the next 2 sections. Even if you do not host a signin widget, your application will still be affected in the ways described (session APIs, token renew)

Reply:
And if you DO host a widget, I think it will only be affected for certain flows (which use iframe). If you provide a `clientId` and `redirectUri` it should do a round-trip redirect to Okta and return to your app as described here: https://github.com/okta/okta-signin-widget#oidc-login-flow-using-pkce-proof-key-for-code-exchange. This **should** work even with 3rd party cookies blocked.

Reply:
it's broken at token renewal even PKCE flow.

Reply:
To cover all cases, changed to: "This issue will not affect your organization if you use the Okta-hosted sign-in page and your app does not call the Sessions API from the browser", as suggested by John.